src/components/ProjectCard.tsx

```diff
@@ -1,82 +1,33 @@
-import { BigNumber } from '@ethersproject/bignumber'
 import * as constants from '@ethersproject/constants'
 import { Skeleton } from 'antd'
 import { PV_V2 } from 'constants/pv'
 import { useProjectHandleText } from 'hooks/useProjectHandleText'
-import { useProjectMetadata } from 'hooks/useProjectMetadata'
-import useSubgraphQuery from 'hooks/useSubgraphQuery'
-import { ProjectTagName } from 'models/project-tags'
-import { Project } from 'models/subgraph-entities/vX/project'
 import Link from 'next/link'
 import { isHardArchived } from 'utils/archived'
 import { formatDate } from 'utils/format/formatDate'
 import { v2v3ProjectRoute } from 'utils/routes'
 
+import { useProjectMetadata } from 'hooks/useProjectMetadata'
+import { DBProject } from 'models/dbProject'
 import { ArchivedBadge } from './ArchivedBadge'
-import ETHAmount from './currency/ETHAmount'
 import Loading from './Loading'
 import ProjectLogo from './ProjectLogo'
 import { ProjectTagsList } from './ProjectTags/ProjectTagsList'
+import ETHAmount from './currency/ETHAmount'
 
-export type ProjectCardProject = Pick<
-  Project,
-  'id' | 'handle' | 'volume' | 'createdAt' | 'terminal' | 'projectId' | 'pv'
-> & { tags?: ProjectTagName[] | null; metadataUri: string | null }
-
-function useProjectCardData(project?: ProjectCardProject | BigNumber) {
```

Comment:
these metadata properties can now all be retrieved from the db (and because we had 0 instances of passing a BigNumber `project` argument to a card), we don't need this hook anymore